### PR TITLE
#1586 Add logging to cypress

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "test:e2e:chrome": "./e2e/setup-docker-e2e.sh",
     "test:e2e:vis-reg": "./e2e/setup-vis-reg-e2e.sh",
     "test:e2e:cypress": "CYPRESS_RETRIES=2 cypress run --config testFiles=\"**/*.spec.e2e.ts\"",
-    "test:e2e:cypress:record": "echo build id $CIRCLE_BUILD_NUM$(echo node) && CYPRESS_RETRIES=2 cypress run --config testFiles=\"**/*.spec.e2e.ts\" --record --parallel --group nodeTests --ci-build-id $CIRCLE_BUILD_NUM$(echo node)",
+    "test:e2e:cypress:record": "echo build id $CIRCLE_BUILD_NUM$(echo node) && CYPRESS_RETRIES=2 cypress run --browser chrome --config testFiles=\"**/*.spec.e2e.ts\" --record --parallel --group nodeTests --ci-build-id $CIRCLE_BUILD_NUM$(echo node)",
```
Comment (desfero) [inline packages/web/package.json]: Is that possible for consistency to use also `--browser chrome` for `io` build?

File: packages/web/cypress/plugins/index.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ const webpack = require("@cypress/webpack-preprocessor");
 require("dotenv").config();
 
 module.exports = on => {
+  require("cypress-log-to-output").install(on, (type, event) => {
```
Comment (desfero) [inline packages/web/cypress/plugins/index.js]: also, I'm just thinking, maybe it's worth to turn on `DevConsoleLogger` over `noopLogger` for e2e tests so then we can have a full view of what's going under the hood? @Moejoe90?

Comment (Moejoe90) [reply packages/web/cypress/plugins/index.js]: Yeah we can try it out and see how much usefull information we can get

File: packages/web/package.json (hunk 1 above)
Comment (drakkein) [reply packages/web/package.json]: it is done already :)

**Comment by Moejoe90** [comment]
Temporary closing this for now

**Pull request closed without merging**